dmutils/logging.py

```diff
@@ -113,6 +113,8 @@ def configure_handler(handler, app, formatter):
     handler.addFilter(AppNameFilter(app.config['DM_APP_NAME']))
     handler.addFilter(RequestExtraContextFilter())
     handler.addFilter(AppStackLocationFilter("app_", app.root_path))
+    if app.config.get('DM_ENVIRONMENT') == 'production':
+        handler.addFilter(AppInstanceFilter())
```

Comment:
Why are we restricting this to production only?

Reply:
Ahh, good point, I think we did this initially just so the tests would pass as this environment variable is not set. But we should run it on any PaaS environment, so I guess for should be for any value other than local?

Reply:
Are we unable to check for the var directly with `os.environ.get('...')` here then?

Reply:
good point @katstevens have updated to do it this way now.